package.json

```diff
@@ -65,6 +65,7 @@
   "devDependencies": {
     "@babel/core": "^7.17.5",
     "@babel/eslint-parser": "^7.17.0",
+    "@carbon/component-usage-report": "0.1.0",
```

Comment:
I think we should possibly rename this to `@carbon/ibm-products-component-usage-report`. It is a bit longer, but I can see there being confusion with the current package name, as there is no mention that it checks for `@carbon/ibm-products` components.

Reply:
It could at some point add in Carbon components, perhaps even allow for others in the same way DevTools does.